src/qt/paymentrequestplus.h

```diff
@@ -13,6 +13,17 @@
 #include <QList>
 #include <QString>
 
+// Used in getMerchant() to return state
+enum GetMerchantStatus {
+    PR_NOTINITIALIZED = 0,
```

Comment:
A bitfield would only make sense if flags can be combined. But IMO things like `PR_UNAUTHENTICATED`together with `PR_AUTHENTICATED` does not make sense. Agree with @laanwj to use plain enumeration in this case.